### DISCUSSION TITLE
Deta 部署出错，提示 No such project

### DISCUSSION THREAD

**Discussion by inchei** [walinejs/waline#1786] [Q&A] [answered]
已通过快速上手设置成功，后想由 Vercel 改为 Deta 部署，按文档操作后，下方步骤在 Creating New Micro 时提示 No such project 并中止。怎么办？

**Comment by MrL646** [comment]
Deta Cloud已停止支持 https://github.com/walinejs/waline/discussions/1822

**Comment by lizheming** [comment, marked as answer]
Deta 现在升级到 Deta Space 了，不支持老的安装方式了。使用 Deta Space 可以非常方便的安装 Waline。
https://deta.space/discovery/@lizheming/waline
点击这个界面的 Install 即可。
